### PR TITLE
For local backend address, use HTTP as default schema

### PR DESCRIPTION
**Pull request by JLXIA**

### CODE REVIEW

**Comment by JLXIA** [comment]
retest

File: tests/start_proxy/start_proxy_test.py [hunk 1]
```diff
@@ -49,8 +49,8 @@ def test_gen_proxy_config(self):
             (['--service=test_bookstore.gloud.run', '--version=2019-11-09r0',
               '--backend=grpc://127.0.0.1:8000', '--http_request_timeout_s=10',
               '--log_jwt_payloads=aud,exp', '--disable_tracing', '--healthz=/'],
-             ['bin/configmanager', '--logtostderr', '--backend_address', 'grpc://127.0.0.1:8000',
-              '--rollout_strategy', 'fixed',  '--healthz', '/',
+             ['bin/configmanager', '--logtostderr', '--rollout_strategy', 'fixed',
```
Comment (nareddyt) [inline tests/start_proxy/start_proxy_test.py:52]: Just curious, what tool did you use to format the python file?

Comment (JLXIA) [reply tests/start_proxy/start_proxy_test.py:52]: manually

**Pull request merged**